static/app/components/events/eventReplay/replayClipPreviewPlayer.tsx

```diff
@@ -0,0 +1,364 @@
+  return 'none';
+}
+
+function ReplayPreviewPlayer({
+  replayId,
+  fullReplayButtonProps,
+  replayRecord,
+  issueCategory,
+  handleBackClick,
+  handleForwardClick,
+  overlayText,
+  showNextAndPrevious,
+  onClickNextReplay,
+}: {
+  replayId: string;
+  replayRecord: ReplayRecord;
+  fullReplayButtonProps?: Partial<ComponentProps<typeof LinkButton>>;
+  handleBackClick?: () => void;
+  handleForwardClick?: () => void;
+  issueCategory?: IssueCategory;
+  onClickNextReplay?: () => void;
+  overlayText?: string;
+  showNextAndPrevious?: boolean;
+}) {
+  const routes = useRoutes();
+  const location = useLocation();
+  const organization = useOrganization();
+  const [isSidebarOpen, setIsSidebarOpen] = useState(true);
+  const {replay, currentTime, isFinished, isPlaying} = useReplayContext();
+  const eventView = EventView.fromLocation(location);
+
+  const fullscreenRef = useRef(null);
+  const {toggle: toggleFullscreen} = useFullscreen({
+    elementRef: fullscreenRef,
+  });
+  const isFullscreen = useIsFullscreen();
+
+  const startOffsetMs = replay?.getStartOffsetMs() ?? 0;
+  const isRageClickIssue = issueCategory === IssueCategory.REPLAY;
+
+  const fullReplayUrl = {
+    pathname: normalizeUrl(`/organizations/${organization.slug}/replays/${replayId}/`),
+    query: {
+      referrer: getRouteStringFromRoutes(routes),
+      t_main: isRageClickIssue ? TabKey.BREADCRUMBS : TabKey.ERRORS,
+      t: (currentTime + startOffsetMs) / 1000,
+      f_b_type: isRageClickIssue ? 'rageOrDead' : undefined,
+    },
+  };
+
+  return (
+    <PlayerPanel>
+      {replayRecord && (
+        <ReplayCellNoPadding
+          key="session"
+          replay={replayRecord}
+          eventView={eventView}
+          organization={organization}
+          referrer="issue-details-replay-header"
+        />
+      )}
+      <PreviewPlayerContainer ref={fullscreenRef} isSidebarOpen={isSidebarOpen}>
+        <PlayerBreadcrumbContainer>
+          <PlayerContextContainer>
+            {isFullscreen ? (
+              <ContextContainer>
+                <ReplayCurrentUrl />
+                <BrowserOSIcons />
+                <ReplaySidebarToggleButton
+                  isOpen={isSidebarOpen}
+                  setIsOpen={setIsSidebarOpen}
+                />
+              </ContextContainer>
+            ) : null}
+            <StaticPanel>
+              <ReplayPlayer
+                overlayText={overlayText}
+                onClickNextReplay={onClickNextReplay}
+              />
+            </StaticPanel>
+          </PlayerContextContainer>
+          {isFullscreen && isSidebarOpen ? <Breadcrumbs /> : null}
+        </PlayerBreadcrumbContainer>
+        <ErrorBoundary mini>
+          <ButtonGrid>
+            {showNextAndPrevious && (
+              <Button
+                size="sm"
+                title={t('Back')}
+                icon={<IconPrevious />}
+                onClick={() => handleBackClick?.()}
+                aria-label={t('Back')}
```

Comment:
I would relabel this to be "Previous clip" or something. Back is like navigation, not a playlist type of thing.

"next" could be updated to match "Next clip"